xc7/primitives/slicem/slicem.pb_type.xml

```diff
@@ -256,9 +256,8 @@
         <direct name="D_SRL.O5_to_SLICEM_MODES.DO5" input="DSRL.O5" output="SLICEM_MODES.DO5"/>
         <direct name="D_SRL.O6_to_SLICEM_MODES.DO6" input="DSRL.O6" output="SLICEM_MODES.DO6"/>
 
-        <!-- WE/CE inputs -->
+        <!-- CE input. Don't connect WE to force routing through WE -->
         <direct name="SLICEM_MODES.CE_to_WE_MUX.CE" input="SLICEM_MODES.CE" output="WE_MUX.CE"/>
-        <direct name="SLICEM_MODES.WE_to_WE_MUX.WE" input="SLICEM_MODES.WE" output="WE_MUX.WE"/>
```

Comment:
Why force `CE` over `WE` here?

Reply:
This is to maintain compatibility with Vivado which cannot route through `WE` to an SRL.

There is no such limitation in hardware, both `CE` and `WE` work. However, when trying to import disassembled fasm with routing through `WE`, Vivado refuses to use that input which results in "antenna net" error.

I added more elaborate comment to the file.

Reply:
You have misinterpreted the error.  Vivado can route using the `WE` to the SRL, but will generally refuse if the `CE` port is available.  The example to check is have the SRL CE and FF CE use different net's, but force them to pack into the same `SLICEM`.

Reply:
Ok, I'll check that tomorrow. But still, if I enable `WE` there will be issues with importing the VPR routing back to Vivado in case when there is only SRL in a slice.

Reply:
The solution to this problem is have VPR choose CE over WE, either by making sure it is the default (e.g. first in the `<mux>`) or by some timing pressure.  Presumably Vivado chooses CE over WE for that reason.

Reply:
I've re-enabled the WE input of WEMUX.

Actually, the `CE` input should be the default as it is defined in the XML:
```
<mux name="WE_MUX" input="WE_MUX.CE WE_MUX.WE" output="WE_MUX.WE_OUT">
  <metadata>
    <meta name="fasm_mux">
      WE_MUX.CE = WEMUX.CE
      WE_MUX.WE = NULL
    </meta>
  </metadata>
</mux>
```